user-guide/Troubleshooting/Known_issues/KI_Upgrade_fails_VerifyElasticStorageType_prerequisite.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: KI_Upgrade_fails_VerifyElasticStorageType_prerequisite
+---
+
+# Upgrade fails because of VerifyElasticStorageType.dll prerequisite
+
+## Affected versions
+
+DataMiner 10.4.0 or newer.
+
+## Cause
+
+Starting from DataMiner version 10.4.0, XML storage for profiles and resources is no longer supported. The _VerifyElasticStorageType_ prerequisite ensures that the system has successfully migrated its data to an indexing database. This verification process includes the following checks to confirm that XML storage is no longer in use:
```

Comment:
Is this really correct? We can not be sure the migration was successful. The user could have just clicked the switch storage type button and maybe didn't migrate data at all?

Reply:
Done